app/packages/state/src/hooks/useExpandSample.ts

```diff
@@ -14,70 +14,81 @@ import * as viewAtoms from "../recoil/view";
 import { LookerStore, Lookers } from "./useLookerStore";
 import useSetExpandedSample from "./useSetExpandedSample";
 
+const setModalFilters = async ({ snapshot, set }: CallbackInterface) => {
+  const paths = await snapshot.getPromise(
+    schemaAtoms.labelPaths({ expanded: false })
+  );
+  const filters = await snapshot.getPromise(filterAtoms.filters);
+  const modalFilters = Object.fromEntries(
+    Object.entries(filters).filter(
+      ([path]) =>
+        paths.some((p) => path.startsWith(p)) || path === "_label_tags"
```

Comment:
I think this can use a nice unit test to make sure we lock down the logic and kind of document one-offs like `_lable_tags`

Reply:
I will follow up this week